sdk/core/core-http/src/policies/systemErrorRetryPolicy.ts

```diff
@@ -88,7 +88,8 @@ export class SystemErrorRetryPolicy extends BaseRequestPolicy {
   public sendRequest(request: WebResourceLike): Promise<HttpOperationResponse> {
     return this._nextPolicy
       .sendRequest(request.clone())
```

Comment:
why do we clone at the beginning of the policy? I don't think this is needed?

Reply:
All the retry policy use pattern of either sending a clone, or saving a clone to use in retries.  Could it be possible that some pipeline might mutate requests thus we always want to send original requests in retries?

Reply:
Hm yeah, maybe it's worried about some downstream policy mutating it.... but it feels wasteful that on a normal successful request (which hopefully is very common!) we're doing a bunch of work to clone to the request just to throw away the original. 

I think it's fine to leave this for now, but I am going to think about it for the 2.x refactor.